docs/sql/scalar.txt

```diff
@@ -835,6 +835,99 @@ Examples
     +---------------------+--------------+
     SELECT 5 rows in set (... sec)
 
+
+Array Functions
+=================
+
+array_merge(first_array, second_array, [ third_array , ... ]) returns array
+-------------------------------------------------------------------
+
+The ``array_merge`` function merges one or more arrays into one array
+
+::
+
+    cr> select array_merge([1,2,3],[3,4,5,6],[7,9,10]) from sys.cluster;
+    +-------------------------------------------+
+    | array_merge([1,2,3], [3,4,5,6], [7,9,10]) |
+    +-------------------------------------------+
+    | [1, 2, 3, 3, 4, 5, 6, 7, 9, 10]           |
+    +-------------------------------------------+
+    SELECT 1 row in set (0.004 sec)
+
+
+It can be used to append elements to array fields
+
+    cr> create table array_merge_example (list array(integer));
+    CREATE OK (0.384 sec)
+    cr> insert into array_merge_example (list) values ([1,2,3]);
+    INSERT OK, 1 row affected (0.303 sec)
+    cr> update array_merge_example set list = array_merge(list,[4,5,6]);
+    UPDATE OK, 1 row affected (0.372 sec)
+    cr> select * from array_merge_example;
+    +--------------------+
+    | list               |
+    +--------------------+
+    | [1, 2, 3, 4, 5, 6] |
+    +--------------------+
+    SELECT 1 row in set (0.037 sec)
+
+array_unique(first_array, [ second_array , ... ]) returns array
+-------------------------------------------------------------------
+
+The ``array_unique`` function merges one or more arrays into one array with unique elements
+
+::
+
+    cr> select array_unique([1,2,3],[3,4,5,6],[7,9,10]) from sys.cluster;
+    +--------------------------------------------+
+    | array_unique([1,2,3], [3,4,5,6], [7,9,10]) |
+    +--------------------------------------------+
+    | [1, 2, 3, 4, 5, 6, 7, 9, 10]               |
+    +--------------------------------------------+
+    SELECT 1 row in set (0.007 sec)
+
+
+It can be used with one argument
+
+    cr> select array_unique([1,2,3,3,4,5,6,7,5,6,9,10]) from sys.cluster;
+    +------------------------------------------+
+    | array_unique([1,2,3,3,4,5,6,7,5,6,9,10]) |
+    +------------------------------------------+
+    | [1, 2, 3, 4, 5, 6, 7, 9, 10]             |
+    +------------------------------------------+
+    SELECT 1 row in set (0.009 sec)
+
+array_subtract(first_array, second_array, [ third_array , ... ]) returns array
+-------------------------------------------------------------------
+
+The ``array_subtract`` function subtracts elements from the first array
```

Comment:
... removes elements from the first array that are contained in the second and further arrays ...